src/SWP/Bundle/TemplateEngineBundle/Resources/config/doctrine/ContainerData.orm.yml

```diff
@@ -1,6 +1,6 @@
 SWP\Bundle\TemplateEngineBundle\Model\ContainerData:
     type: entity
-    table: null
+    table: swp_container_data
```

Comment:
To easily locate the SWP tables when needed. Note that some vendors can create tables too, prefixes makes it more clear to see what is part of the SWP and what's not when it comes to the database. It's a good practice. There is also a possibility to create a custom listener that will apply the prefix automatically so we could make it also configurable if needed, but I don't think it's a blocker :)